test/index.js

```diff
@@ -272,3 +273,26 @@ describe('.replacePaths', () => {
         results.includes('test/font.woff2').should.be.false();
     });
 });
+
+describe('.refresh', () => {
+    it('should empty the cache of file hashes after a refresh()', () => {
+        fs.writeFileSync(path.join(__dirname, 'variable_file.txt'), 'File Content 1');
+
+        const staticifyObj = staticify(ROOT);
+        const versionedPath = staticifyObj.getVersionedPath('/test/variable_file.txt');
+        const versioned = versionedPath.split('.');
```

Comment:
Any reason you are splitting this here?

Reply:
No particular reason expect for the fact that I copied it from the `.getVersionedPath` test.